accounts/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from django.views.generic import TemplateView
+
+from . import views
+
+
+urlpatterns = [
+    path('signup/', views.SignUp.as_view(), name='signup'),
```

Comment:
I think that we should have a sign in view as well.

Reply:
Done